virtwho/manager/subscriptionmanager/subscriptionmanager.py

```diff
@@ -73,6 +75,33 @@ def readConfig(self):
         self.cert_file = os.path.join(consumerCertDir, cert)
         self.key_file = os.path.join(consumerCertDir, key)
 
+    def _check_owner_lib(self, config):
+        """
+        Try to check values of env and owner. These values has to be
+        equal to values obtained from Satellite server.
+        :param config: Configuration of virt-who
+        :return: None
+        """
+        if config is None:
+            return None
+
```

Comment:
I would continue with this method only if 'rhsm_username', 'rhsm_password' are not set and 'owner' and 'env' are set.